controllers/errors.controller.js

```diff
@@ -0,0 +1,23 @@
+exports.handlePsqlErrors = ((err,request,response,next)=>{
```

Comment:
If you have trouble getting error handlers to fire correctly consider swapping the order of psql errors and customErrors. As this is still read in thread the order does matter.